src/shared/parseReleaseNotes.ts

```diff
@@ -6,36 +6,57 @@
  */
 
 import { marked } from 'marked';
+import * as semver from 'semver';
 
 const parseReleaseNotes = (notes: string, version: string, baseUrl: string): marked.Token[] => {
   let found = false;
+  let findClosest = false;
```

Comment:
I think you can delete this var entirely.  

then, line 72 can just ask if `closestVersion` ever got defined

Reply:
Good call 🎉